ccx_upgrade_risk_data_eng_tests.sh

```diff
@@ -22,14 +22,10 @@ PATH_TO_LOCAL_DATA_ENG_SERVICE=${PATH_TO_LOCAL_DATA_ENG_SERVICE:="../ccx-upgrade
 #set NOVENV is current environment is not a python virtual env
 [ "$VIRTUAL_ENV" != "" ] || NOVENV=1
 
-function install_reqs() {
-    pip install -r requirements.txt
-}
-
 function prepare_venv() {
     echo "Preparing environment"
     # shellcheck disable=SC1091
-    virtualenv -p python3 venv && source venv/bin/activate && pip install -r requirements/upgrades_data_eng_service.txt
+    virtualenv -p python3 venv && source venv/bin/activate && pip install -r requirements.txt
```

Comment:
If they are installed a Dockerfile level, do we still need this? Why not setting the venv in the dockerfile?